infra/gp-grafana-instance/templates/argocd-notifications-patch.yaml

```diff
@@ -1,30 +1,11 @@
 apiVersion: redhatcop.redhat.io/v1alpha1
 kind: Patch
-metadata:
-  name: patch-argocd-notifications
-spec:
-  serviceAccountRef:
-    name: patch-operator-sa
-  patches:
-    patch-argocd-notifications:
-      targetObjectRef:
-        apiVersion: argoproj.io/v1alpha1
-        kind: ArgoCD
-        name: {{ .Values.argocd.name }}
-        namespace: {{ .Values.argocd.namespace }}
-      patchTemplate: |
-        spec:
-          notifications:
-            enabled: {{ .Values.argocd.notifications.enabled }}
-      patchType: application/merge-patch+json
----
-apiVersion: redhatcop.redhat.io/v1alpha1
-kind: Patch
 metadata:
   name: patch-argocd-notifications-configmap
+  namespace: syn-patch-operator
```

Comment:
bitte nicht hardcoden. vor allem wenn die werte schon in den values definiert sind.